Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1296,16 +1321,18 @@ extension PurchasesOrchestrator: StoreKit2PurchaseIntentListenerDelegate {
         // stop the compiler from checking availability in the functions.
         // We also need to ensure that we're on Xcode >= 15.3, since that is when
         // PurchaseIntents were first made available on macOS.
-        #if !os(tvOS) && !os(watchOS) && compiler(>=5.10)
+        #if compiler(>=5.10) && !os(tvOS) && !os(watchOS) && !os(visionOS)
```

Comment:
~~Just checking but what's the reason we're not making this available in visionOS? If it's only for the tracking, we should probably just wrap that and avoid removing functionality?~~ Edit: Just saw the comment in the previous PR. I think it's ok then, just a bit nervous to change this in the tracking PR 😅.

Reply:
Yeah... I don't like it much either but [there's an explanation](https://github.com/RevenueCat/purchases-ios/pull/4888#discussion_r1998802411) 😄